src/funding/status-card/status-card.html

```diff
@@ -0,0 +1,36 @@
+<template>
+  <pcard>
+    <header>
+      <div class="daoName">
+        <dao-icon-name primary-dao.to-view="dao"></dao-icon-name>
+      </div>
+      <pchip color="${chipColor}"> ${status} </pchip>
+    </header>
+    <div class="content">
+      <div class="token-progress" repeat.for="token of dao.tokens" if.to-view="!swapCompleted">
+        <img src="${token.logoURI}" />
+        <div>
+          <div class="progress-bar-content">
+            <div>
+              ${token.deposited | ethwei:token.decimals}
```

Comment:
Is best to use the formatted-number component when displaying bignumbers.  Standard/default format is two visible decimals.  And there will automatically be a tooltip showing the full precision.

Reply:
Changing this now